news/2018-11-01-beatmap-spotlights-summer-2018.md

```diff
@@ -0,0 +1,314 @@
+
+[![](/wiki/shared/news/2018-11-01-beatmap-spotlights-summer-2018/osu/glitch.jpg)](http://osu.ppy.sh/beatmapsets/785774)
+
+Coming straight from the Mapping Guild with a Featured Artist song, [Mao](https://osu.ppy.sh/users/2204515), [Gero](https://osu.ppy.sh/users/1467715), [Mirash](https://osu.ppy.sh/users/2841009) and [Noffy](https://osu.ppy.sh/users/1541323) proved us that beatmapping is truly a piece of art. With a mixture of streams and slider velocity changes, accompanied by one of the most stunning storyboard, this beatmap is definitely worth being up in this summer chart.
+
+[![](/wiki/shared/news/2018-11-01-beatmap-spotlights-summer-2018/osu/negai.jpg)](https://osu.ppy.sh/beatmapsets/671268)
+
+Lightly edited from the original version made five years ago, [Hollow Wings](https://osu.ppy.sh/users/416662) brings us a pleasant mapset that has aged very well with Ishida Yoko's "Negai". Built on a low-bpm swing beat, this map mixes slow and simple sections with faster, energetic choruses that will have your cursor dancing across the screen to their lovely melody. 
+
+Together with [Pata-Mon](https://osu.ppy.sh/users/6149313) and [Suzuki_1112](https://osu.ppy.sh/users/3170678) they created a very enjoyable set to play.
+
+[![](/wiki/shared/news/2018-11-01-beatmap-spotlights-summer-2018/osu/messiah.jpg)](http://osu.ppy.sh/beatmapsets/793438)
+
+[eiri-](https://osu.ppy.sh/users/3388410)'s strikingly original set of Liz Triangle's "Messiah" has 3 unique difficulties that all express the song quite beautifully. [Jounzan](https://osu.ppy.sh/users/1848318)'s top diff has great intensity and spacing contrast, along with a unique use of flow. [Mirash](https://osu.ppy.sh/users/2841009) and [Delis](https://osu.ppy.sh/users/1603923)' collab gives us a nicely improvised approach to emphasis, providing a fun gameplay experience. 
+
+Last but not least, Vass' Desire by [Vass_Bass](https://osu.ppy.sh/users/188248) displays a very natural movement around the playfield, which is more easily playable, yet displays a high attention to detail. With a nice range of mapping styles, it's easy to say something in this set for everybody!
+
+[![](/wiki/shared/news/2018-11-01-beatmap-spotlights-summer-2018/osu/unknown-x-known.jpg)](http://osu.ppy.sh/beatmapsets/800070)
+
+When considering versatility in mapping skills, [Lasse](https://osu.ppy.sh/users/896613) is pretty much out of the question. Their rendition of Meramipop's "Unknown x known" remixed by DYES IWASAKI definitely shows it too, with its great cursor control and aesthetics. Using heavy slider velocity manipulation to control cursor momentum, it matches the ups and downs of the song perfectly. This map is sure to not disappoint you, and is definitely a welcome one to this season's Spotlights!
```

Comment:
When considering versatility in mapping skills, [Lasse](https://osu.ppy.sh/users/896613) is pretty much out of the question

"out of the question" in this context means they wouldn't be considered, aka Lasse would not be one of them :v